apps/arweave/src/ar_inflation.erl

```diff
@@ -50,40 +50,30 @@ calculate_post_15_y1_extra() ->
 pre_15_calculate(Height) when Height =< ?REWARD_DELAY ->
 	1;
 pre_15_calculate(Height) ->
-	?WINSTON_PER_AR
-		* 0.2
-		* ?GENESIS_TOKENS
-		* math:pow(2, -(Height - ?REWARD_DELAY) / ?PRE_15_BLOCK_PER_YEAR)
-		* math:log(2)
-        / ?PRE_15_BLOCK_PER_YEAR.
+	?WINSTON_PER_AR * 0.2 * ?GENESIS_TOKENS *
```

Comment:
Ditto earlier comment. I'd remove elvis constraints on this sort of thing - long equations are hard to parse and genearlly the engineer writing/maintaining the code will know what formatting is the most intelligible . Something that will be hard for a pattern matching engine to get right

Reply:
this rule for spaces around operators can be solved in other ways than I did it, I mode more use of the line here which I can adjust and make these operations be performed on individual lines

Reply:
it's now newlined for each math operation